mysql-test/main/partition_error.test

```diff
@@ -16,7 +16,7 @@ let $MYSQLD_DATADIR= `SELECT @@datadir`;
 --echo #
 CREATE TABLE t1 (a int);
 CREATE OR REPLACE VIEW v1 AS SELECT * FROM t1;
---error ER_CHECK_NO_SUCH_TABLE
+--error ER_WRONG_OBJECT
 ALTER TABLE t1 EXCHANGE PARTITION p0 WITH TABLE v1;
 DROP VIEW v1;
 DROP TABLE t1;
```

Comment:
The test case above does two wrong things. One is that `t1` is a non-partitioned table. The other is that `v1` is a view, not a physical table. So, I'd suggest adding another simpler test case in which everything is correct except the target table is a view.

Reply:
Hi added another test case.